### PR TITLE
re-arrange key provision docs

### PR DESCRIPTION
**Pull request by somerobmitchell**

### CODE REVIEW

File: docs/source/VUDRM-key-provision.md [hunk 1]
```diff
@@ -1,8 +1,236 @@
...
+
+### Endpoints
+
+#### Default
+
+Get a CPIX 2.1 document with all DRM systems `<client-name>` is entitled to use.
+
+##### Optional query params:
+
+* `drm` : comma separated list of DRM systems to be included included in the response.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-key-provision.md]:
```suggestion
* `drm` : comma separated list of DRM systems to be included in the response.
```

File: docs/source/VUDRM-key-provision.md [hunk 2]
```diff
@@ -1,8 +1,236 @@
...
+</cpix:CPIX>
+```
+
+#### Decrypt
+
+Get a CPIX 2.1 document with only a single content key.
+
+##### Optional query params:
+
+* `drm` : comma separated list of DRM systems to be included included in the response.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-key-provision.md]:
```suggestion
* `drm` : comma separated list of DRM systems to be included in the response.
```

File: docs/source/VUDRM-key-provision.md [hunk 3]
```diff
@@ -1,8 +1,236 @@
...
+</cpix:CPIX>
+```
+
+#### Multikey
+
+Get a CPIX 2.1 document that uses separate keys for video and audio tracks. Only supported by DASH with PlayReady or Widevine.
+
+##### Optional query params:
+
+* `drm` : comma separated list of DRM systems to be included included in the response.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-key-provision.md]:
```suggestion
* `drm` : comma separated list of DRM systems to be included in the response.
```

File: docs/source/VUDRM-key-provision.md [hunk 4]
```diff
@@ -1,8 +1,236 @@
...
+    <cpix:ContentKeyUsageRule kid="22222222-2222-2222-22222-22222222222">
+      <cpix:AudioFilter></cpix:AudioFilter>
+    </cpix:ContentKeyUsageRule>
+  </cpix:ContentKeyUsageRuleList>
+</cpix:CPIX>
+```
+
+#### Multikey with audio clear
+
+Get a CPIX 2.1 document that use only 1 content key for video; audio tracks are left on the clear. Only supported by DASH with PlayReady or Widevine.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-key-provision.md]:
```suggestion
Get a CPIX 2.1 document that uses only 1 content key for video; audio tracks are left on the clear. Only supported by DASH with PlayReady or Widevine.
```

File: docs/source/VUDRM-key-provision.md [hunk 5]
```diff
@@ -1,8 +1,236 @@
...
+
+##### Optional query params:
+
+* `drm` : comma separated list of DRM systems to be included included in the response.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-key-provision.md]:
```suggestion
* `drm` : comma separated list of DRM systems to be included in the response.
```

File: docs/source/VUDRM-key-provision.md [hunk 6]
```diff
@@ -1,8 +1,236 @@
...
+</cpix:CPIX>
+```
+
+#### Keys
+
+Get encryption keys and license URLs for a given `<content-id>` (JSON response).
+
+##### Optional query params:
+
+* `drm` : comma separated list of DRM systems to be included included in the response.
```
Comment (PerritonBranch) [inline docs/source/VUDRM-key-provision.md]:
```suggestion
* `drm` : comma separated list of DRM systems to be included in the response.
```

File: docs/source/VUDRM-key-provision.md [hunk 7]
```diff
@@ -1,8 +1,236 @@
...
+    <cpix:ContentKeyUsageRule kid="22222222-2222-2222-22222-22222222222">
+      <cpix:AudioFilter></cpix:AudioFilter>
+    </cpix:ContentKeyUsageRule>
+  </cpix:ContentKeyUsageRuleList>
+</cpix:CPIX>
+```
+
+#### Multikey with audio clear
+
+Get a CPIX 2.1 document that uses only 1 content key for video; audio tracks are left on the clear. Only supported by DASH with PlayReady or Widevine.
```
Comment (fschurer) [inline docs/source/VUDRM-key-provision.md]: typo (audio tracks are left in the clear, not on the clear)

File: docs/source/VUDRM-key-provision.md [hunk 8]
```diff
@@ -422,9 +652,9 @@ The values are:
 
 ## Unified Streaming Integration
 
-The encryption keys provided by the Key Provider API are compatible with [Unified Streaming Platform's](https://www.unified-streaming.com/) mp4split product.
+The encryption keys provided by the Key Provider APIs are compatible with [Unified Streaming Platform's](https://www.unified-streaming.com/) mp4split product.
 
-The recommended approach is to call the Key Provider API and retrieve CENC and Fairplay Keys. Once the encryption keys have been retrieved they can be used with mp4split to generate an ism:
+The recommended approach is to call the Key Provider APIs and retrieve CENC and Fairplay Keys. Once the encryption keys have been retrieved they can be used with mp4split to generate an ism:
```
Comment (PerritonBranch) [inline docs/source/VUDRM-key-provision.md]:
```suggestion
The recommended approach is to call the CPIX Key Provider API to retrieve CENC and Fairplay Keys. Once the encryption keys have been retrieved they can be used with mp4split to generate an ism:
```

**Pull request merged**